server/src/main/java/io/crate/execution/dml/upsert/TransportShardUpsertAction.java

```diff
@@ -247,33 +255,78 @@ private static String getExceptionMessage(Throwable e) {
     @Override
     protected WriteReplicaResult<ShardUpsertRequest> processRequestItemsOnReplica(IndexShard indexShard, ShardUpsertRequest request) throws IOException {
         Translog.Location location = null;
-        for (ShardUpsertRequest.Item item : request.items()) {
-            if (item.seqNo() == SequenceNumbers.SKIP_ON_REPLICA || item.source() == null) {
-                if (logger.isTraceEnabled()) {
```

Comment:
Why did we skip this?

Reply:
Skip what? The logic is the same as before

Reply:
the log trace entry.

Reply:
Ah right, added it back

Reply:
thx!